portfolio/src/main/webapp/script.js

```diff
@@ -22,6 +21,47 @@ var counter = 1;
 var intervalLength = setInterval(change, 2000);
 document.getElementById('commentBtn').addEventListener('click', getComments);
 
+var script = document.createElement('script');
+script.src = `https://maps.googleapis.com/maps/api/js?key=${MAPKEY}&callback=initMap`;
```

Comment:
I'm assuming MAPKEY is meant to be replaced by the developer using this?

Reply:
yup!